pkg/apiserver/certificate/cacert_controller.go

```diff
@@ -67,6 +69,9 @@ var (
 	optionalMutationWebhooks = []string{
 		"labelsmutator.antrea.io",
 	}
+	conversionWebhooks = []string{
```

Comment:
nit: to be accurate it seems that the name should be something like `crdsWithConversionWebhooks`?